codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -272,6 +275,13 @@ private boolean readClientPrefaceString(ByteBuf in) throws Http2Exception {
             if (bytesRead == 0 || !ByteBufUtil.equals(in, in.readerIndex(),
                                                       clientPrefaceString, clientPrefaceString.readerIndex(),
                                                       bytesRead)) {
+                String http1Request =
+                    in.toString(in.readerIndex(), Math.min(in.readableBytes(), 1024), CharsetUtil.US_ASCII);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Unnecessary use of fully qualified name 'Math.min' due to existing static import 'java.lang.Math.min' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUnnecessaryFullyQualifiedName)